### ISSUE TITLE
Textos para página de upload

### ISSUE THREAD

**Issue by Gschwend** [linea-it/pz-server#49]
**1. Basic Information**

   Please provide the basic information about the new data product. 

**2. Upload Files** --> levar os 3 parágrafos para a parte de cima. Remover pequena descrição em cima de cada botão de upload

   Please select the file(s) for the upload. The main file is the one containing the data. It must be a single file. For example, if the product type is Spec-z Sample, Training Set, or Validation Set, the data must be tabular, and the tool supports the formats: CSV, FITS, HDF5, and parquet. Otherwise, if the product type is Validation Results or Photo-z Table, the file format is free. Please provide them compressed in a single .tar file in case of multiple files.  

 The description file is supposed to contain relevant information about the data product. It can be a PDF document, a Jupyter notebook, etc. 

The Auxiliary Files are in free format and can be multiple files (press the upload button as many times as necessary). 

**3. Association Columns**

Please associate the column names of your file with those expected by the tool. 

**4. Confirm**

Please review your information before pressing the FINISH button.
